ghost/core/test/unit/frontend/helpers/social_url.test.js

```diff
@@ -1,11 +1,11 @@
-const should = require('should');
 const handlebars = require('../../../../core/frontend/services/theme-engine/engine').handlebars;
 const helpers = require('../../../../core/frontend/services/helpers');
 const social_url = require('../../../../core/frontend/helpers/social_url');
 
 const socialData = {
     facebook: 'testuser-fb',
     twitter: 'testuser-tw',
+    x: 'testuser-tw',
```

Comment:
Whilst adding the new property here is correct with the new shape of the data, supporting x in social_urls will require additional changes to the helper (you'll need to map x to twitter) - I _think_ it makes sense to do those updates separately and focus in this PR on getting the API changes complete.

Reply:
I'll add these changes (tests) in a separate PR.

However, we can avoid making any changes to the `social_urls` template helper by creating an alias in the `@tryghost/social-urls` library in the SDK doc as I've done in this [PR](https://github.com/TryGhost/SDK/pull/512/files).

All that was needed for it work was to have `x` as a property in the user / author api and having `@tryghost/social-urls` export a function called `x`.